code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -66,6 +66,11 @@ God bless America.
 	RefreshParts()
 	fry_loop = new(list(src), FALSE)
 
+/obj/machinery/deepfryer/process()
+	if(prob(100))
```

Comment:
Im sorry, if(prob(100))?
Thats basically just if(TRUE)

Reply:
LOL I forgot to change it

Reply:
I set it to 100 to test it- adjusted it to what it should be